docs/src/components/banner/AppHeaderBanner.tsx

```diff
@@ -1,24 +1,34 @@
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
+import { Theme } from '@mui/material/styles';
 import { Link } from '@mui/docs/Link';
 import ROUTES from 'docs/src/route';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
 
+const linkStyleOverrides = (theme: Theme) => ({
+  color: 'inherit',
+  textDecorationColor: 'currentColor',
+  '&:hover': {
+    color: (theme.vars || theme).palette.primary[200],
+  },
+  ...theme.applyDarkStyles({
+    color: 'inherit',
+    '&:hover': {
+      color: (theme.vars || theme).palette.primary[200],
+    },
+  }),
+});
+
```

Comment:
This change is required as the styles were broken, probably because of the introduction of `applyDarkStyles`